google-cloud-storage/src/main/java/com/google/cloud/storage/ParallelCompositeUploadBlobWriteSessionConfig.java

```diff
@@ -231,7 +241,31 @@ public ParallelCompositeUploadBlobWriteSessionConfig withPartCleanupStrategy(
         executorSupplier,
         bufferAllocationStrategy,
         partNamingStrategy,
-        partCleanupStrategy);
+        partCleanupStrategy,
+        partMetadataFieldDecorator);
+  }
+
+  /**
+   * Specify a Part Metadata Field decorator, this will manipulate the metadata associated with part
+   * files, the ultimate object metadata will remain unchanged.
+   *
+   * <p><i>Default: </i> {@link PartMetadataFieldDecorator#noOp()}
+   *
+   * @since 2.35.1 This new api is in preview and is subject to breaking changes.
```

Comment:
```suggestion
   * @since 2.36.0 This new api is in preview and is subject to breaking changes.
```

Since this is a feature, it will trigger a minor version not a patch.